models/place.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+// mongoose.connect('mongodb://localhost/ibi-ironhack',  {useMongoClient: true});
+// mongoose.Promise = require('bluebird');
+
+const placeSchema = new Schema({
+    name: {type: String},
+    location: {
```

Comment:
This is not the proper way to define the schema for coordinates in Mongooose, instead use:

```javascript
location: {
  type: {type: String, default: "Point"},
  coordinates: [Number]
},
```